src/Cms/Blueprint.php

```diff
@@ -341,20 +342,10 @@ public static function load(string $name): array
 	{
 		$props = static::find($name);
 
-		$normalize = function ($props) use ($name) {
-			// inject the filename as name if no name is set
-			$props['name'] ??= $name;
-
-			// normalize the title
-			$title = $props['title'] ?? ucfirst($props['name']);
+		// inject the filename as name if no name is set
+		$props['name'] ??= $name;
 
-			// translate the title
-			$props['title'] = I18n::translate($title, $title);
```

Comment:
@distantnative What was the reason for removing these lines? This is the cause of the #5086 issue.

Reply:
I made this change in another PR as well cause it looked like it just does the same again in the constructor, but maybe not